liquibase-core/src/test/java/liquibase/sqlgenerator/core/CreateViewGeneratorTest.java

```diff
@@ -1,23 +1,34 @@
 package liquibase.sqlgenerator.core;
 
+import liquibase.database.Database;
 import liquibase.database.core.MSSQLDatabase;
+import liquibase.database.core.OracleDatabase;
 import liquibase.sql.Sql;
 import liquibase.sqlgenerator.SqlGeneratorChain;
 import liquibase.statement.core.CreateViewStatement;
+import liquibase.util.SqlParser;
+import liquibase.util.StringClauses;
 import org.junit.Before;
 import org.junit.Test;
+import org.mockito.Mock;
+import org.mockito.Mockito;
 
 import java.util.TreeSet;
 
 import static org.junit.Assert.assertEquals;
 
 public class CreateViewGeneratorTest {
+    private static final String LSP = System.lineSeparator();
 
     private CreateViewGenerator generator;
+    @Mock
+    Database database;
```

Comment:
No need to mock it, you can just create it via new OracleDatabase(); as part of the call in every method

Reply:
done